### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Dmytro-Kretov**

### CODE REVIEW

File: src/transformState.js [hunk 1]
```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  for (const object of actions) {
```
Comment (nazarmatsevych) [inline src/transformState.js:8]:
```suggestion
  for (const action of actions) {
```

File: src/transformState.js [hunk 2]
```diff
@@ -5,7 +5,25 @@
...
+    switch (object.type) {
```
Comment (nazarmatsevych) [inline src/transformState.js:9]: Use destructuring of object properties

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Good job,
pls check my comments

File: src/transformState.js [hunk 3]
```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  for (const action of actions) {
```
Comment (AlisaVlasova) [inline src/transformState.js:8]:
```suggestion
  for (const action of actions) {
    const { type, extraData , keysToRemove } = action;
    
    switch (type) {
```
this destructuring was meant)

**Review by AlisaVlasova — APPROVED** [review]

**Pull request closed without merging**